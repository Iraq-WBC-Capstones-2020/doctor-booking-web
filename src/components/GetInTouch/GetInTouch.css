.getIntouch {
  background: #f5f9fd;
}
