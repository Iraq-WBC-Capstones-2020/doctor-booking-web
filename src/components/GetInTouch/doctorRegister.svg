<svg width="528" height="279" viewBox="0 0 528 279" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M520.62 266.269H6.7364C3.02759 266.269 0 269.097 0 272.615C0 276.109 3.00236 278.962 6.7364 278.962H520.62C524.329 278.962 527.356 276.133 527.356 272.615C527.331 269.097 524.329 266.269 520.62 266.269Z" fill="#DBE8D2"/>
<path d="M109.598 170.168V147.729C109.598 144.14 112.701 141.216 116.511 141.193C162.707 141.05 191.722 141.05 194.068 141.193C194.043 141.193 193.765 141.169 193.387 141.097C194.648 136.058 190.612 131.233 185.137 131.209C175.196 131.162 155.592 131.162 116.486 131.281C106.899 131.304 99.1025 138.673 99.1025 147.705V170.144C99.1025 179.224 106.924 186.593 116.562 186.593H192.605V176.681H116.536C112.701 176.705 109.598 173.781 109.598 170.168Z" fill="#194657"/>
<path d="M29.4922 171.784V180.032C29.4922 188.209 40.442 194.817 53.9148 194.817H151.126C164.624 194.817 175.548 188.185 175.548 180.032V171.784H29.4922Z" fill="#14353F"/>
<path d="M96.2864 41.0009H26.5004C17.3924 41.0009 10 47.9654 10 56.5463V149.51C10 158.091 17.3924 165.055 26.5004 165.055H96.2864C105.394 165.055 112.787 158.091 112.787 149.51V56.5463C112.812 47.9654 105.42 41.0009 96.2864 41.0009Z" fill="#68698C"/>
<path d="M104.033 41H34.2465C25.1385 41 17.7461 47.9645 17.7461 56.5454V149.509C17.7461 158.09 25.1385 165.054 34.2465 165.054H104.033C113.141 165.054 120.533 158.09 120.533 149.509V56.5454C120.558 47.9645 113.166 41 104.033 41Z" fill="#6397FF"/>
<path d="M187.266 157.115H23.5993C20.3446 157.115 17.7207 159.587 17.7207 162.653C17.7207 165.72 20.3446 168.192 23.5993 168.192H187.266C190.521 168.192 193.145 165.72 193.145 162.653C193.145 159.611 190.521 157.115 187.266 157.115Z" fill="#8889C2"/>
<path d="M92.3509 162.653C92.3509 159.587 94.9748 157.115 98.2295 157.115H23.5993C20.3446 157.115 17.7207 159.587 17.7207 162.653C17.7207 165.72 20.3446 168.192 23.5993 168.192H98.2042C94.9748 168.192 92.3509 165.72 92.3509 162.653Z" fill="#68698C"/>
<path d="M18.0884 170.168V147.729C18.0884 144.14 21.1917 141.216 25.0014 141.193C71.1975 141.05 100.212 141.05 102.558 141.193C102.533 141.193 102.256 141.169 101.877 141.097C103.139 136.058 99.1018 131.233 93.6269 131.209C83.6863 131.162 64.0826 131.162 24.9762 131.281C15.3888 131.304 7.59277 138.673 7.59277 147.705V170.144C7.59277 179.224 15.4141 186.593 25.0519 186.593H101.095V176.681H25.0267C21.1917 176.705 18.0884 173.781 18.0884 170.168Z" fill="#194657"/>
<path d="M102.357 186.616H94.6113V266.269H102.357V186.616Z" fill="#14353F"/>
<path d="M123.701 259.019H71.2479C69.1286 259.019 67.3877 260.635 67.3877 262.655C67.3877 264.652 69.1033 266.292 71.2479 266.292H123.701C125.82 266.292 127.561 264.676 127.561 262.655C127.561 260.635 125.845 259.019 123.701 259.019Z" fill="#14353F"/>
<path d="M211.552 69.6222C213.924 68.2911 216.699 67.0788 219.399 67.6493C219.954 68.7427 218.717 69.8599 217.607 70.5016C214.958 72.0704 212.309 73.6392 209.685 75.208C209.357 75.3982 209.004 75.6121 208.852 75.9449C208.348 76.9195 209.609 77.8227 210.391 78.6071C212.107 80.3661 211.577 83.3611 209.912 85.1676C208.247 86.9741 205.774 87.9249 203.403 88.8043C202.545 89.1134 201.637 89.4461 200.754 89.2322C196.666 88.2339 197.146 82.1488 198.76 79.7005C201.46 75.5884 207.238 72.0704 211.552 69.6222Z" fill="#FFB8B8"/>
<path d="M151.934 44.3313C151.934 44.3313 169.444 84.4071 174.919 87.9012C180.394 91.3954 199.114 78.0843 199.114 78.0843L203.429 90.3495C203.429 90.3495 163.742 114.5 161.017 110.815C158.292 107.131 144.996 83.4325 144.996 83.4325C144.996 83.4325 145.122 46.5419 151.934 44.3313Z" fill="white"/>
<path d="M173.026 96.4102C171.36 97.0045 169.569 97.5749 167.803 97.2659C166.239 96.9807 164.927 96.0299 163.741 95.0316C161.521 93.1775 159.502 91.1096 157.762 88.8514C156.298 86.9498 154.835 84.8106 152.463 84.0737C150.319 83.4319 147.972 84.1926 146.231 85.5474C149.562 91.4661 158.821 107.748 161.041 110.767C163.766 114.451 203.453 90.3014 203.453 90.3014L201.536 84.8343C192.226 89.1367 182.714 93.0111 173.026 96.4102Z" fill="#DEF2FA"/>
<path d="M158.216 143.855C162.833 141.193 247.404 164.986 253.938 174.185C260.498 183.384 273.416 259.447 273.416 259.447L265.696 258.853L229.188 186.593L175.574 170.239C175.574 170.239 104.552 174.851 158.216 143.855Z" fill="#1F1F3A"/>
<path d="M97.4873 158.901C97.4873 158.901 116.384 176.657 140.681 186.617C164.977 196.576 191.595 199.904 191.595 199.904C191.595 199.904 197.146 233.467 226.034 256.214H235.066L213.595 174.209L154.936 154.48L158.241 143.879L132.279 134.204C132.279 134.204 90.3472 131.756 97.4873 158.901Z" fill="#2A284F"/>
<path d="M151.934 44.3307C151.934 44.3307 98.9764 35.2744 76.875 50.4158L94.3341 146.707L158.216 143.855C158.216 143.855 160.31 68.8611 151.934 44.3307Z" fill="#6397FF"/>
<path d="M113.18 74.7561C112.625 63.7507 112.146 52.7691 111.742 41.7637C98.7486 42.2867 85.4776 44.521 76.8994 50.3921L94.3333 146.707L126.981 145.257C123.348 137.889 120.648 130.068 118.731 122.153C114.971 106.631 113.987 90.6581 113.18 74.7561Z" fill="#BEB8FF"/>
<path d="M123.172 40.3377C123.172 40.3377 142.019 138.031 153.599 144.592C165.18 151.152 247.228 172.83 257.269 182.386C257.269 182.386 256.79 172.616 247.682 166.27C238.574 159.923 162.556 138.531 161.774 138.364C161.017 138.174 158.166 51.4144 151.934 44.331C151.934 44.331 127.587 40.3852 123.172 40.3377Z" fill="white"/>
<path d="M146.106 63.6558C145.576 62.1346 134.5 59.3297 134.5 59.3297C134.5 59.3297 142.195 57.5945 143.432 56.7388C144.365 56.1208 141.665 47.1358 140.202 42.5007C133.314 41.4549 125.493 40.3615 123.172 40.3377C123.172 40.3377 130.64 78.9636 139.042 109.27C141.186 96.6482 146.535 64.8681 146.106 63.6558Z" fill="#DEF2FA"/>
<path d="M105.965 41.6451C105.965 41.6451 113.332 147.967 124.559 151.628C135.786 155.288 187.609 186.45 191.595 199.88L117.242 202.02C117.242 202.02 107.983 176.111 95.04 166.246C85.1498 158.711 67.0599 75.6596 74.8308 51.3907C77.2024 44.022 90.9779 40.1 105.965 41.6451Z" fill="white"/>
<path d="M86.1345 97.6458C87.5979 94.6746 89.0864 91.6796 89.7424 88.4469C90.8525 82.8135 89.1117 76.7285 85.1506 72.3786C84.6207 71.7843 83.99 71.2139 83.2079 70.9762C81.8959 70.5959 80.5083 71.3327 79.3729 72.0458C77.2284 73.4007 75.1847 74.8982 73.2168 76.5145C73.8223 87.0445 75.4118 99.0958 77.5563 110.909C78.1366 110.339 78.6665 109.768 79.1963 109.15C82.123 105.704 84.1414 101.663 86.1345 97.6458Z" fill="#DEF2FA"/>
<path d="M94.5106 152.436C95.3685 150.558 95.9235 148.514 95.7974 146.493C95.5703 142.999 93.3753 139.838 90.6505 137.461C88.0265 135.155 84.898 133.444 81.9209 131.566C84.4691 142.025 87.2696 151.247 89.9188 157.641C91.9876 156.619 93.5519 154.527 94.5106 152.436Z" fill="#DEF2FA"/>
<path d="M96.756 12.5504C97.4372 11.1005 99.2538 9.62675 98.295 8.31941C97.9923 7.91533 97.4625 7.65386 97.2606 7.20224C96.7812 6.18014 98.2446 5.30066 99.4304 5.18181C101.348 4.96788 103.316 5.3482 105.031 6.20391C104.88 4.92034 104.148 3.70808 103.063 2.89991C105.385 2.06797 108.109 2.35321 110.178 3.63678C109.977 2.70976 109.548 1.83027 108.967 1.06964C111.49 0.760632 114.165 1.54504 116.032 3.18515C116.511 2.11551 116.183 0.760632 115.25 0C119.261 0.546704 122.465 3.63678 124.257 7.05962C126.578 11.5521 126.78 17.4232 123.323 21.2264C120.346 24.4828 110.405 28.3573 106.066 25.7902C104.073 24.6255 103.24 21.9633 101.625 20.3945C99.4808 18.3503 95.1161 16.0208 96.756 12.5504Z" fill="#2A284F"/>
<path d="M125.443 10.9579C125.443 10.9579 129.606 30.3778 124.282 31.3286L120.977 31.1622L123.021 45.1151C123.021 45.1151 113.862 47.6584 108.766 45.4954L108.337 29.6885C108.337 29.6885 104.527 27.1927 104.325 23.8887C104.123 20.5847 106.798 22.415 108.74 22.6289L109.901 25.9804L111.743 24.2928C111.743 24.2928 111.062 16.8528 110.809 16.4488C110.557 16.0209 115.578 14.2144 115.578 14.2144C115.578 14.2144 122.289 14.4521 125.443 10.9579Z" fill="#FFB8B8"/>
<path d="M117.621 33.7769C116.562 33.0876 115.78 31.9467 115.679 30.7106C117.041 30.8057 118.378 30.9008 119.741 30.9959C120.144 31.0196 120.573 31.0672 120.952 31.2098C120.952 31.2098 120.977 31.2098 120.977 31.2336L121.507 34.799C121.431 34.799 121.33 34.8228 121.255 34.8228C119.943 34.7515 118.681 34.4663 117.621 33.7769Z" fill="#2A284F"/>
<path d="M90.2719 144.164C90.3476 144.924 90.2719 145.709 90.4737 146.445C90.6755 147.23 91.1549 147.919 91.6343 148.585C92.1137 149.25 92.6435 149.94 93.4256 150.201C94.4348 150.558 95.5954 150.106 96.327 149.369C97.0587 148.632 97.3867 147.634 97.639 146.636C98.1436 144.686 98.295 142.666 98.1436 140.693C98.0679 139.742 97.8913 138.72 97.2606 137.96C96.7812 137.365 96.0748 136.961 95.3683 136.605C94.3087 136.058 93.249 135.559 92.1641 135.107C91.054 134.632 89.8934 134.204 88.6571 134.18C80.306 134.061 90.0196 141.787 90.2719 144.164Z" fill="#FFB8B8"/>
<path d="M88.5561 53.2918C85.6547 47.0403 76.4457 46.4699 72.6865 52.3172C59.1632 73.4247 28.1808 123.531 36.1535 128.095C46.2707 133.895 87.8749 142.88 87.8749 142.88L91.6846 133.824L63.1748 112.288L92.9461 77.7508C92.9461 77.7508 93.4507 63.798 88.5561 53.2918Z" fill="white"/>
<path d="M272.559 257.664C272.559 257.664 287.697 263.44 293.626 269.074C293.626 269.074 266.15 268.907 265.444 266.982C264.737 265.056 265.721 258.876 265.721 258.876C265.721 258.876 270.742 257.664 272.559 257.664Z" fill="#2A284F"/>
<path d="M234.738 255.002C234.738 255.002 248.009 263.44 253.938 269.074C253.938 269.074 226.463 268.907 225.756 266.982C225.05 265.057 226.034 256.19 226.034 256.19C226.034 256.19 232.039 253.291 234.738 255.002Z" fill="#2A284F"/>
<path d="M85.9071 52.6265C85.5286 53.0543 85.1249 53.5535 85.2006 54.124C85.2763 54.7182 85.8566 55.0985 86.3864 55.4313C88.4301 56.6911 90.4989 57.9509 92.5425 59.2345C92.9967 59.5197 93.5013 59.8525 93.6274 60.3754C93.7788 61.0647 93.1481 61.6828 92.5678 62.1344C90.1205 64.0597 87.6732 65.9851 85.2259 67.9104C95.5449 82.8616 106.015 98.1456 113.61 114.404C108.816 82.4575 107.63 41.6449 107.63 41.6449C104.981 41.3596 100.717 41.2646 98.2193 41.3359C94.0816 45.0677 89.5402 48.5143 85.9071 52.6265Z" fill="#DEF2FA"/>
<path d="M287.445 44.7355H236.203C231.611 44.7355 227.902 48.2296 227.902 52.5557V130.948C227.902 135.274 231.611 138.769 236.203 138.769H287.445C292.037 138.769 295.746 135.274 295.746 130.948V52.5557C295.746 48.2296 292.037 44.7355 287.445 44.7355Z" fill="#BEB8FF"/>
<path d="M295.241 44.7344H240.795C237.086 44.7344 234.084 47.563 234.084 51.0571V132.445C234.084 135.939 237.086 138.767 240.795 138.767H295.241C298.95 138.767 301.953 135.939 301.953 132.445V51.0334C301.953 47.563 298.95 44.7344 295.241 44.7344Z" fill="#DEF2FA"/>
<path d="M244.024 83.0747C242.46 79.8182 241.249 76.2765 241.678 72.7111C242.132 69.1694 244.605 65.6515 248.288 64.7958C251.24 64.1064 254.318 65.1998 256.942 66.6023C261.055 68.8366 264.637 71.9505 267.286 75.6348C269.204 71.6414 272.585 68.1711 276.95 66.6498C281.314 65.1285 286.613 65.8178 289.943 68.8841C292.945 71.6414 294.005 75.8962 293.475 79.8182C292.945 83.7402 291.028 87.377 288.808 90.7285C284.367 97.4078 275.612 110.291 265.798 106.06C256.236 101.948 248.263 91.8933 244.024 83.0747Z" fill="#EC5946"/>
<path d="M269.986 77.3933H265.243V98.5484H269.986V77.3933Z" fill="white"/>
<path d="M278.844 90.2123V85.7436H256.389V90.2123H278.844Z" fill="white"/>
<path d="M214.566 254.637L224.355 254.637L224.355 133.364L214.566 133.364L214.566 254.637Z" fill="#14353F"/>
<path d="M124.717 254.637L134.506 254.637L134.506 133.364L124.717 133.364L124.717 254.637Z" fill="#14353F"/>
<path d="M394.974 255.159L404.764 255.159L404.764 133.886L394.974 133.886L394.974 255.159Z" fill="#14353F"/>
<path d="M195.867 251.754L205.656 251.754L205.656 130.481L195.867 130.481L195.867 251.754Z" fill="#14353F"/>
<path d="M106.203 140.327L420.038 140.327L420.038 127.468L106.203 127.468L106.203 140.327Z" fill="#194657"/>
<path d="M106.206 140.327L243.28 140.327L243.28 127.468L106.206 127.468L106.206 140.327Z" fill="#9B3930"/>
</svg>
