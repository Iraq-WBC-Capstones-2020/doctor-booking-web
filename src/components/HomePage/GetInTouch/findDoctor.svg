<svg width="464" height="356" viewBox="0 0 464 356" fill="none" xmlns="http://www.w3.org/2000/svg">
<g clip-path="url(#clip0)">
<path d="M373.589 0.213379H0.15918V241.983H373.589V0.213379Z" fill="#E6E6E6"/>
<path d="M362.912 30.5378H10.8379V227.68H362.912V30.5378Z" fill="white"/>
<path d="M373.43 0H0V16.0377H373.43V0Z" fill="#6397FF"/>
<path d="M11.789 11.1717C13.413 11.1717 14.7294 9.84091 14.7294 8.19927C14.7294 6.55762 13.413 5.22681 11.789 5.22681C10.1651 5.22681 8.84863 6.55762 8.84863 8.19927C8.84863 9.84091 10.1651 11.1717 11.789 11.1717Z" fill="white"/>
<path d="M22.9502 11.1717C24.5741 11.1717 25.8906 9.84091 25.8906 8.19927C25.8906 6.55762 24.5741 5.22681 22.9502 5.22681C21.3262 5.22681 20.0098 6.55762 20.0098 8.19927C20.0098 9.84091 21.3262 11.1717 22.9502 11.1717Z" fill="white"/>
<path d="M34.1113 11.1717C35.7352 11.1717 37.0517 9.84091 37.0517 8.19927C37.0517 6.55762 35.7352 5.22681 34.1113 5.22681C32.4874 5.22681 31.1709 6.55762 31.1709 8.19927C31.1709 9.84091 32.4874 11.1717 34.1113 11.1717Z" fill="white"/>
<path d="M133.295 50.993H42.8281V207.225H133.295V50.993Z" fill="#E6E6E6"/>
<path d="M224.3 50.993H148.911V93.9977H224.3V50.993Z" fill="#6397FF"/>
<path d="M224.3 105.974H148.911V152.244H224.3V105.974Z" fill="#E6E6E6"/>
<path d="M224.3 164.22H148.911V207.225H224.3V164.22Z" fill="#E6E6E6"/>
<path d="M330.92 50.993H240.454V119.078H330.92V50.993Z" fill="#E6E6E6"/>
<path d="M330.92 139.14H240.454V207.225H330.92V139.14Z" fill="#6397FF"/>
<path d="M427.24 121.178C421.272 110.897 409.466 110.418 409.466 110.418C409.466 110.418 397.962 108.917 390.582 124.576C383.703 139.171 374.21 153.263 389.053 156.679L391.735 148.17L393.395 157.313C395.509 157.468 397.629 157.505 399.746 157.424C415.643 156.9 430.782 157.577 430.295 151.76C429.647 144.028 432.982 131.071 427.24 121.178Z" fill="#2F2E41"/>
<path d="M370.65 176.072L365.43 189.557L339.331 214.182C339.331 214.182 323.671 225.909 330.631 229.426C337.591 232.944 343.97 216.528 343.97 216.528L371.81 199.525L377.61 180.176L370.65 176.072Z" fill="#FFB8B8"/>
<path d="M379.35 325.581L384.569 339.066H392.109L385.729 323.236L379.35 325.581Z" fill="#FFB8B8"/>
<path d="M401.969 331.444L407.189 344.93H414.728L408.349 329.099L401.969 331.444Z" fill="#FFB8B8"/>
<path d="M401.389 333.789L411.249 331.444L398.489 279.263L402.726 239.112L402.839 239.1C402.839 239.1 410.089 210.664 404.289 203.042C398.489 195.42 369.49 198.352 369.49 198.352C369.49 198.352 358.47 209.492 359.05 234.703C359.63 259.914 359.05 281.608 363.11 291.575C367.17 301.542 378.77 327.926 378.77 327.926L388.05 326.754L378.19 283.367L376.614 245.651C377.868 260.765 380.393 289.424 381.67 291.575C383.41 294.507 401.389 333.789 401.389 333.789Z" fill="#2F2E41"/>
<path d="M430.968 188.385L435.028 203.042L453.007 235.876C453.007 235.876 468.667 249.947 462.287 251.706C455.907 253.465 446.048 238.807 446.048 238.807L425.748 210.078L420.528 188.971L430.968 188.385Z" fill="#FFB8B8"/>
<path d="M410.088 146.757C418.096 146.757 424.588 140.194 424.588 132.099C424.588 124.004 418.096 117.441 410.088 117.441C402.081 117.441 395.589 124.004 395.589 132.099C395.589 140.194 402.081 146.757 410.088 146.757Z" fill="#FFB8B8"/>
<path d="M402.549 139.721L399.069 150.861L393.27 163.76L414.729 154.965C414.729 154.965 413.569 145.584 414.729 143.239L402.549 139.721Z" fill="#FFB8B8"/>
<path d="M427.488 163.173L414.149 152.033L399.069 158.483L399.66 148.969L383.57 154.164L381.09 154.965L377.61 173.141C377.61 173.141 375.29 174.899 372.97 184.867C371.63 190.724 369.686 196.423 367.17 201.87C367.17 201.87 392.109 191.902 404.289 205.388C404.289 205.388 406.029 204.215 405.449 202.456C404.869 200.697 410.669 191.902 410.669 191.902C410.669 191.902 410.669 193.661 412.989 190.144C415.309 186.626 417.049 186.039 417.049 186.039L427.488 163.173Z" fill="#6397FF"/>
<path d="M384.57 156.724L382.036 154.659C382.036 154.659 379.93 153.792 378.19 156.724C376.45 159.656 367.75 177.831 367.75 177.831L378.19 183.108L384.57 156.724Z" fill="#6397FF"/>
<path d="M419.368 162.587L427.488 163.173C427.488 163.173 432.128 165.519 432.128 169.036C432.128 172.554 433.288 191.316 433.288 191.316L420.528 195.42L415.309 179.004L419.368 162.587Z" fill="#6397FF"/>
<path d="M382.893 337.014L373.034 341.118C373.034 341.118 361.434 341.705 363.174 347.568C364.914 353.431 382.313 349.327 382.313 349.327C382.313 349.327 397.973 349.913 397.973 345.809C397.973 341.705 395.653 337.014 393.333 337.014H382.893Z" fill="#2F2E41"/>
<path d="M404.933 342.291L395.074 346.395C395.074 346.395 383.474 346.981 385.214 352.845C386.954 358.708 404.353 354.603 404.353 354.603C404.353 354.603 420.013 355.19 420.013 351.086C420.013 346.981 417.693 342.291 415.373 342.291H404.933Z" fill="#2F2E41"/>
<path d="M424.601 120.187L412.937 113.957L396.831 116.506L393.498 131.513L401.794 131.189L404.111 125.674V131.098L407.939 130.948L410.16 122.169L411.549 131.513L425.157 131.23L424.601 120.187Z" fill="#2F2E41"/>
<path d="M346.588 48.984C334.985 39.0873 320.788 32.8129 305.723 30.9243C290.658 29.0357 275.374 31.6144 261.732 38.3466C248.091 45.0787 236.678 55.6741 228.885 68.8433C221.091 82.0126 217.252 97.188 217.834 112.523C218.416 127.858 223.395 142.691 232.165 155.217C240.934 167.744 253.116 177.423 267.229 183.078C281.341 188.733 296.775 190.12 311.653 187.069C326.53 184.019 340.21 176.663 351.027 165.896L447.083 247.831C447.808 248.449 448.645 248.917 449.548 249.208C450.451 249.498 451.402 249.606 452.346 249.526C453.291 249.445 454.21 249.177 455.052 248.737C455.894 248.297 456.642 247.693 457.253 246.961C457.864 246.229 458.327 245.382 458.614 244.469C458.902 243.556 459.008 242.595 458.928 241.64C458.848 240.686 458.583 239.756 458.147 238.905C457.712 238.054 457.115 237.298 456.391 236.681L360.335 154.745C371.493 138.555 376.269 118.737 373.726 99.1727C371.183 79.6087 361.505 61.71 346.588 48.984ZM340.395 146.855C332.912 155.821 322.967 162.341 311.818 165.593C300.669 168.844 288.816 168.681 277.759 165.122C266.702 161.564 256.937 154.772 249.699 145.603C242.461 136.435 238.074 125.302 237.095 113.614C236.115 101.925 238.586 90.2056 244.196 79.9369C249.805 69.6681 258.3 61.3114 268.608 55.9236C278.915 50.5357 290.571 48.3587 302.103 49.6678C313.634 50.9769 324.522 55.7133 333.391 63.2781C345.283 73.4222 352.702 87.9265 354.016 103.6C355.329 119.274 350.43 134.833 340.395 146.855Z" fill="#3F3D56"/>
<path opacity="0.3" d="M257.72 153.936C246.464 144.334 239.193 130.805 237.348 116.034C235.504 101.264 239.222 86.3321 247.764 74.2043C246.642 75.3464 245.558 76.5424 244.51 77.7925C239.541 83.7451 235.78 90.6289 233.443 98.0508C231.105 105.473 230.237 113.287 230.887 121.049C231.537 128.81 233.693 136.366 237.232 143.285C240.771 150.203 245.623 156.35 251.512 161.373C257.401 166.396 264.21 170.197 271.552 172.56C278.895 174.922 286.625 175.8 294.302 175.142C301.98 174.484 309.454 172.304 316.298 168.726C323.142 165.148 329.222 160.242 334.19 154.289C335.235 153.037 336.219 151.756 337.142 150.443C326.753 160.996 312.818 167.213 298.105 167.86C283.392 168.507 268.976 163.536 257.72 153.936Z" fill="black"/>
</g>
<defs>
<clipPath id="clip0">
<rect width="463.816" height="356" fill="white"/>
</clipPath>
</defs>
</svg>
