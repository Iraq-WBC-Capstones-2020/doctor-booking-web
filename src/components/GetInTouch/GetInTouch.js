import React from 'react';
import { Row, Col, Button, Container } from 'react-bootstrap';
import findeDoctor from './findDoctor.svg';
import doctorRegister from './doctorRegister.svg';
import { Link } from 'react-router-dom';

function GetInTouch() {
  return (
    <div className="getIntouch my-5 py-5" id="getInTouch">
      <Container>
        <Row className="mt-4">
          <Col lg="6" className="">
            <img
              src={findeDoctor}
              className="img-fluid IntroSection__image"
              alt=""
            />
          </Col>
          <Col lg="6" className=" ">
            <div>
              <h1 className="my-3">
                Find a doctor and <span className="d-block">make an</span>{' '}
                apoinnment online
              </h1>
              <p className="textShow">
                search for the best doctor for you and book an appointment at
                the time that fits your time
              </p>
              <div className="my-4">
                <Link to="/search">
                  <Button variant="primary">Find Doctor</Button>
                </Link>
              </div>
            </div>
          </Col>
        </Row>
        <Row className=" mt-5 mb-4">
          <Col lg="6">
            <div>
              <h1 className="my-3">
                Are you Doctor?
                <span className="d-block">Register here</span>
              </h1>
              <p className="textShow">
                With our website you are be more time manageable <br /> and you
                be comfortable about setting appoinments and dealing with your
                vistors
              </p>
              <div className="my-4">
                <Link to="/register">
                  <Button variant="primary">Register</Button>
                </Link>
              </div>
            </div>
          </Col>
          <Col lg="6" className=" ">
            <img
              src={doctorRegister}
              className="img-fluid IntroSection__image"
              alt=""
            />
          </Col>
        </Row>
      </Container>
    </div>
  );
}

export default GetInTouch;
