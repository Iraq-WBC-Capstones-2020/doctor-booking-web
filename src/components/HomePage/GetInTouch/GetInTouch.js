import React from 'react';
import { Row, Col, Button, Container } from 'react-bootstrap';
import { Link } from 'react-router-dom';
import findeDoctor from './findDoctor.svg';
import doctorRegister from './doctorRegister.svg';
import '../HomePageGlobalStyle.scss';
import { useTranslation } from 'react-i18next';

function GetInTouch() {
  const { t, i18n } = useTranslation(['common', 'validation']);
  return (
    <div id="getInTouch" className="getIntouch my-5 py-5">
      <Container>
        <Row className="">
          <Col lg="6" className="">
            <img
              src={findeDoctor}
              className="img-fluid IntroSection__image"
              alt=""
            />
          </Col>
          <Col lg="6">
            <div>
              <h1 className="my-3 sectionHeading">
                {t('validation:homePage.findDoc.headLine.title1')}{' '}
                <span className="d-block">
                  {' '}
                  {t('validation:homePage.findDoc.headLine.title2')}
                </span>{' '}
                {t('validation:homePage.findDoc.headLine.titleSpan')}
              </h1>
              <p className="sectionCaption">
                {t('validation:homePage.findDoc.paragraph')}
              </p>
              <div className="my-4">
                <Link to="/search">
                  <Button className="btn-lg sectionBtn" variant="primary">
                    {t('common:findDoctor')}
                  </Button>
                </Link>
              </div>
            </div>
          </Col>
        </Row>
        <Row className="my-5">
          <Col lg="6">
            <div>
              <h1 className="my-3 sectionHeading">
                {t('validation:homePage.docRig.headLine.title1')}
                <span className="d-block">
                  {t('validation:homePage.docRig.headLine.titleSpan')}
                </span>
              </h1>
              <p className="sectionCaption">
                {t('validation:homePage.docRig.paragraph')}
              </p>
              <div className="my-4">
                <Link to="/register">
                  <Button className="btn-lg sectionBtn">
                    {t('common:register')}
                  </Button>
                </Link>
              </div>
            </div>
          </Col>
          <Col lg="6" className=" ">
            <img
              src={doctorRegister}
              className="img-fluid IntroSection__image"
              alt=""
            />
          </Col>
        </Row>
      </Container>
    </div>
  );
}

export default GetInTouch;
