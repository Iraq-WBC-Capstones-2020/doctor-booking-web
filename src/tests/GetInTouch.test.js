import React from 'react';
import GetInTouch from '../components/HomePage/GetInTouch/GetInTouch';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

it('GetInTouch render correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <GetInTouch />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
