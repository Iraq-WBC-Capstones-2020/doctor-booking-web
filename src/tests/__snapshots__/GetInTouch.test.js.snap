// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GetInTouch render correctly 1`] = `
<div
  className="getIntouch my-5 py-5"
  id="getInTouch"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-6"
      >
        <img
          alt=""
          className="img-fluid IntroSection__image"
          src="findDoctor.svg"
        />
      </div>
      <div
        className="col-lg-6"
      >
        <div>
          <h1
            className="my-3 sectionHeading"
          >
            validation:homePage.findDoc.headLine.title1
             
            <span
              className="d-block"
            >
               
              validation:homePage.findDoc.headLine.title2
            </span>
             
            validation:homePage.findDoc.headLine.titleSpan
          </h1>
          <p
            className="sectionCaption"
          >
            validation:homePage.findDoc.paragraph
          </p>
          <div
            className="my-4"
          >
            <a
              href="/search"
              onClick={[Function]}
            >
              <button
                className="btn-lg sectionBtn btn btn-primary"
                disabled={false}
                type="button"
              >
                common:findDoctor
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="my-5 row"
    >
      <div
        className="col-lg-6"
      >
        <div>
          <h1
            className="my-3 sectionHeading"
          >
            validation:homePage.docRig.headLine.title1
            <span
              className="d-block"
            >
              validation:homePage.docRig.headLine.titleSpan
            </span>
          </h1>
          <p
            className="sectionCaption"
          >
            validation:homePage.docRig.paragraph
          </p>
          <div
            className="my-4"
          >
            <a
              href="/register"
              onClick={[Function]}
            >
              <button
                className="btn-lg sectionBtn btn btn-primary"
                disabled={false}
                type="button"
              >
                common:register
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="  col-lg-6"
      >
        <img
          alt=""
          className="img-fluid IntroSection__image"
          src="doctorRegister.svg"
        />
      </div>
    </div>
  </div>
</div>
`;
